### PR TITLE
Feature cucumber report, hooks and retray

### PR DESCRIPTION
**Pull request by jpgonzales27**

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -10,5 +10,8 @@
         "overwrite": false,
         "html": false,
         "json": true
+    },
+    "retries": {
+        "runMode": 1
     }
-}
+}
```
Comment (JRojas999) [inline cypress.json]: It should have blank line at the end.

File: cypress/integration/CucumberTest/contacts/contact/contactStep.js [hunk 2]
```diff
@@ -28,5 +31,11 @@ When(/^I create a new Contact with fields$/, function (dataTable) {
...
     validateContact(newObject)
-    // deleteContact()
+    cy.location('pathname').then((url) => {
+        idObject = url.substr(1)
+    })
 });
+
+And(/^I delete "(.*?)"$/, (valor) => {
+    feature.deleteOne(valor, token, idObject)
+});
```
Comment (JRojas999) [inline cypress/integration/CucumberTest/contacts/contact/contactStep.js]: It should have blank line at the end.

File: cypress/integration/contact/create-contact.spec.js [hunk 3]
```diff
@@ -20,17 +31,45 @@ describe('test for contact feature', () => {
...
     });
+
+    after(() => {
+        if (contactId !== '') {
+            feature.deleteOne("Contact", token, contactId)
+        }
+        if (accountId !== '') {
+            feature.deleteOne("Account", token, accountId)
+        }
+        // feature.deleteOne("Contact", token, idObject)
```
Comment (JRojas999) [inline cypress/integration/contact/create-contact.spec.js]: It should be deleted if not necessary.

File: cypress/cucumber-json/contact.cucumber.json [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+            "result": {
+              "status": "passed",
+              "duration": 2046000000
+            }
+          }
+        ]
+      }
+    ]
+  }
+]
```
Comment (JRojas999) [inline cypress/cucumber-json/contact.cucumber.json:117]: It should have blank line at the end.

File: cypress/cucumber-json/empty.cucumber.json [hunk 5]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (JRojas999) [inline cypress/cucumber-json/empty.cucumber.json:1]: It should have blank line at the end.

File: cypress/integration/CucumberTest/contacts/contact/contactStep.js (hunk 2 above)
Comment (jpgonzales27) [reply cypress/integration/CucumberTest/contacts/contact/contactStep.js]: Done

File: cypress.json (hunk 1 above)
Comment (jpgonzales27) [reply cypress.json]: Done

File: cypress/integration/contact/create-contact.spec.js (hunk 3 above)
Comment (jpgonzales27) [reply cypress/integration/contact/create-contact.spec.js]: Done

File: cypress/cucumber-json/contact.cucumber.json (hunk 4 above)
Comment (jpgonzales27) [reply cypress/cucumber-json/contact.cucumber.json:117]: This file is auto generate for cucumber preprocessor

**Review by JRojas999 — APPROVED** [review]

**Review by edsonarios — APPROVED** [review]

**Pull request merged**